eZ/Publish/Core/settings/search_engines/legacy/criterion_handlers_common.yml

```
@@ -36,6 +36,10 @@ parameters:
             - "tab_search_normalize"
 
 services:
+    ezplatform.search.legacy.gateway.criteria_converter.trash:
```

again, this is not search in terms of our domain definition, though indeed feature is called search in trash (which should be trash filtering, but what can I do...).


```suggestion
    ezplatform.trash.search.legacy.gateway.criteria_converter:
```